R/adjective.R

```diff
@@ -39,6 +39,7 @@ adjective <- c(
   "exclusive",
   "exceptional",
   "exciting",
+  "exhilarating"
```

Comment:
Looks like you forgot the trailing comma. I'll suggest that here and you can just accept to update your PR.

```suggestion
  "exhilarating",
```